flink-examples/doc/flink-high-error-count-alert/README.md

```diff
@@ -0,0 +1,151 @@
+
+On the Logstash host, download the plugin gem file from [Logstash Pravega output plugin](https://github.com/pravega/logstash-output-pravega/releases), for example, `logstash-output-pravega-0.3.0.gem`.
+
+Install the plugin, assuming Logstash is installed at `/usr/share/logstash/`
+```
+$ /usr/share/logstash/bin/logstash-plugin install logstash-output-pravega-0.3.0.gem
+```
+
+Copy the contents under flink-examples/doc/flink-high-error-count-alert/filters/ to the Logstash host, e.g., in directory ~/pravega.
+update **pravega_endpoint** in ~/pravega/90-pravega-output.conf
+
+```
+output {
+    pravega {
+        pravega_endpoint => "tcp://127.0.0.1:9090"   <- update to point to your Pravega controller
+        stream_name => "apacheaccess"
+        scope => "examples"
+    }
+}
+```
+
+Start logstash, assuming it is installed at /usr/share/logstash/bin.
+Note that sometimes it may take a minute or two for logstash to start. For troubleshooting, the logstash log files are 
+normally at /var/log/logstash. To restart, type Ctrl-C, then re-run the command.
+
+```
+$ sudo /usr/share/logstash/bin -f ~/pravega
+Sending Logstash's logs to /var/log/logstash which is now configured via log4j2.properties
+```
+
+Normally Logstash is configured to receive data from remote log shippers, such as filebeat. For simplicity in this demo
+Logstash is configured read data from /tmp/access.log.
+
+## Run in Docker Container ##
+
+Create a file at /tmp/access.log
+```
+$ touch /tmp/access.log
+```
+
+Run script below to start container from prebuilt image. Adjust parameters to your need.
+```
+#!/bin/sh
+set -u
+
+PRAVEGA_SCOPE=${PRAVEGA_SCOPE:-examples}
+PRAVEGA_STREAM=${PRAVEGA_STREAM:-apacheaccess}
+CONTAINER_NAME=pravega
+IMAGE_NAME=emccorp/pravega-demo
+
+docker run -d --name $CONTAINER_NAME \
+    -p 9090:9090 \
+    -p 9091:9091 \
+    -p 9600:9600 \
+    -v /tmp/access.log:/opt/data/access.log \
+    -v $PWD/logs/:/var/log/pravega/ \
+    -e PRAVEGA_SCOPE=${PRAVEGA_SCOPE} \
+    -e PRAVEGA_STREAM=${PRAVEGA_STREAM} \
+    ${IMAGE_NAME} 
+```
+
+You can skip **PRAVEGA_SCOPE** and **PRAVEGA_STREAM** if you want to use the defaults.
+
+To check Pravega
+```
+$ curl localhost:9091/v1/scopes
+```
+
+To check Logstash and output plugin for Pravega via Logstash monitoring API running on port 9600. Logstash may take one or two minutes to start as a delay is introduced to wait for Pravega to start.
+```
+# The output should contain the name and the version (your version may differ) of the plugin  
+#    "name" : "logstash-output-pravega",
+#    "version" : "0.3.0.pre.SNAPSHOT"
+
+$ curl localhost:9600/_node/plugins?pretty
+```
+
+The log files for Pravega and Logstash should be in the **logs** directory under current directory if you the script above to start the container.
+
+More details can be found on github [pravega docker](https://github.com/hldnova/pravega-docker) and on dockerhub [pravega docker image](https://hub.docker.com/r/emccorp/pravega-demo/) 
```

Comment:
We have to move this (https://github.com/hldnova/pravega-docker) to samples repository to keep the code and the test environment intact.

Reply:
I am moving docker file to pravega/logstash-output-pravega as it has versions of both pravega and the logstash plugin artifact. See  https://github.com/pravega/logstash-output-pravega/pull/12. Please let me know what you think.